questionnaire/questionnaire-service.js

```diff
@@ -588,6 +588,27 @@ function createQuestionnaireService({
         };
     }
 
+    async function getQuestionnaireIdsBySubmissionStatus(status) {
+        return db.getQuestionnaireIdsBySubmissionStatus(status);
+    }
+
+    async function postSubmissionsByStatus(status) {
```

Comment:
This is the reason I wanted to use the other functionality we chatted about. What I could do here is update the submission to IN_PROGRESS at this point and it would avoid that issue.